Switch Key Vault sub resources to use Key Vault ID instead of URL (#2820)
terraform-providers/terraform-provider-azurerm: website/docs/r/key_vault_secret.html.markdown

* `value` - (Required) Specifies the value of the Key Vault Secret.

> * `vault_id` - (Required) Specifies the ID of the Key Vault instance the certificate resides in, available from the `azurerm_key_vault.id` property.

Reviewer: would it make sense to make this consistent with the other ID fields?

```suggestion
* `vault_id` - (Required) The ID of the Key Vault where the Secret should be created.
```
